pkg/app/api/grpcapi/api.go

```diff
@@ -376,17 +381,62 @@ func (a *API) RequestPlanPreview(ctx context.Context, req *apiservice.RequestPla
 }
 
 func (a *API) GetPlanPreviewResults(ctx context.Context, req *apiservice.GetPlanPreviewResultsRequest) (*apiservice.GetPlanPreviewResultsResponse, error) {
-	_, err := requireAPIKey(ctx, model.APIKey_READ_WRITE, a.logger)
+	key, err := requireAPIKey(ctx, model.APIKey_READ_WRITE, a.logger)
 	if err != nil {
 		return nil, err
 	}
 
-	// TODO: Implement GetPlanPreviewResults RPC.
+	const freshDuration = 24 * time.Hour
```

Comment:
Is it really needed to be in the block? I feel it should be out of there.

Reply:
👍 
The command output contains the diff generated by our drift detector or external tools like terraform.
Of course, we are masking sensitive data while comparing but just in case.
So we can think of this as an additional safety guard.

Reply:
Sorry, what I mean was this `const freshDurations` should be out of this function block.

Reply:
Ah, I misunderstood what you wanted to say.
I think we should put it inside that function to strict its scope. It must only be used by that function, no one else.

Reply:
Fine, I got your point 👍